internal/internal_task_handlers_test.go

```diff
@@ -869,18 +869,11 @@ func (t *TaskHandlersTestSuite) TestWorkflowTask_NondeterministicLogNonexistingI
 	taskHandler := newWorkflowTaskHandler(testDomain, params, nil, t.registry)
 	request, err := taskHandler.ProcessWorkflowTask(&workflowTask{task: task}, nil)
 
-	t.NotNil(request)
-	response := request.(*s.RespondDecisionTaskFailedRequest)
-
-	// NOTE: we might acctually want to return an error
-	// but since previously we checked the wrong error type, it may break existing customers workflow
-	// The issue is that we change the error type and that we change the error message, the customers
-	// are checking the error string - we plan to wrap all errors to avoid this issue in client v2
-	t.NoError(err)
-	t.NotNil(response)
+	t.Nil(request)
+	t.ErrorContains(err, "nondeterministic workflow")
 
 	// Check that the error was logged
-	ignoredWorkflowLogs := logs.FilterMessage("Ignored workflow panic error")
+	ignoredWorkflowLogs := logs.FilterMessage("Illegal state caused panic")
```

Comment:
probably worth changing this var name now too

Reply:
missed that, thanks